<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>数据集位置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_choose_data">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>94</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>数据基本属性</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>大小</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>类别个数</string>
    </property>
   </widget>
   <widget class="QComboBox" name="btn_spin">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>92</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>聚类算法选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_parameter">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>95</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>设置算法参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>16</width>
      <height>471</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>671</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="t_ans">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tmessage_show">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>331</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ldata_shape">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ldata_class">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ldata_name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_show_image">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>510</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>点击查看聚类图形可视化</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>聚类评价指标</string>
    </property>
   </widget>
   <zorder>line_2</zorder>
   <zorder>label</zorder>
   <zorder>btn_choose_data</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>btn_spin</zorder>
   <zorder>label_6</zorder>
   <zorder>btn_set_parameter</zorder>
   <zorder>btn_run</zorder>
   <zorder>line</zorder>
   <zorder>t_ans</zorder>
   <zorder>tmessage_show</zorder>
   <zorder>graphicsView</zorder>
   <zorder>ldata_shape</zorder>
   <zorder>ldata_class</zorder>
   <zorder>ldata_name</zorder>
   <zorder>btn_show_image</zorder>
   <zorder>label_7</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
